tests/CMakeLists.txt

```diff
@@ -31,34 +31,36 @@ add_library(simple-dll SHARED dll.c)
 
 add_executable(
   aarch64-mingw-tests.exe
-  bigdata-test.cpp
+  bigdata-test.c
   chkstk-test.c
-  chkstk-test.cpp
+#  chkstk-test.cpp
   dll-test.c
-  dll-test.cpp
-  iostream-test.cpp
+#  dll-test.cpp
+  gtest_like_c.c
+#  iostream-test.cpp
```

Comment:
The CPP files are redundant now and just add to the maitenance cost.

Reply:
Removed